circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   python:
-    version: 2.7.9
+    version: 3.6.6
```

Comment:
before we commit to this, lets do a quick check to see if anyone is actively using the library under 2.7

I'm sure at this point it might be fine to say that all new releases of django-rq-scheduler will only support and test for 3.6+

If we make that change though, there are a couple of other changes that need to happen. And we'll want to make it clear when we put this out on pypi that 2.7 has been dropped.

Thoughts @tom-price ?

Reply:
@oudeismetis, raising it to 3.6+ may reduce the reach of the project, but I'd trade more approachable to make contributions to for that.
I strongly agree with what you said about making it clear that 2.7 has been dropped on pypi if we proceed.

Reply:
@bashhack as mentioned above I don't feel really that qualified to review these changes.